plugin-mapping.properties

```diff
@@ -121,4 +121,5 @@ seatunnel.sink.IoTDB = connector-iotdb
 seatunnel.sink.Neo4j = connector-neo4j
 seatunnel.sink.FtpFile = connector-file-ftp
 seatunnel.sink.Socket = connector-socket
+seatunnel.sink.datahub = connector-datahub
```

Comment:
`seatunnel.sink.datahub`rename to `seatunnel.sink.DataHub` it's better,We'd better be consistent with pluginname